vendor/assets/javascripts/react_rails_img.js.jsx.erb

```diff
@@ -4,15 +4,10 @@ prefix = Rails.application.config.assets.prefix
 should_compute_assets_path = Rails.env.production? # for performance
 if should_compute_assets_path
   images_dir = Rails.root.join('app/assets/images/').to_s
-  assets.each_file do |file_path|
+  Dir["#{images_dir}/**/*"].each do |file_path|
```

Comment:
using `Dir["#{images_dir}/**/*"]` will not be allowed to use assets placed in vendor gems.

I think a better solution is using following way:

```
images_enumerator = defined?(assets) ? assets.each_file : Dir["#{images_dir}**/*.*"]
images_enumerator.each do |file_path|
  file_name = file_path.sub(images_dir, '')
  file_path = asset_path(file_name)
end
```